pkg/kubelet/dockershim/helpers_linux.go

```diff
@@ -62,7 +62,8 @@ func getSeccompDockerOpts(seccompProfile string) ([]dockerOpt, error) {
 		return nil, fmt.Errorf("unknown seccomp profile option: %s", seccompProfile)
 	}
 
-	fname := strings.TrimPrefix(seccompProfile, "localhost/") // by pod annotation validation, name is a valid subpath
+	// get the full path of seccomp profile when prefixed with 'localhost/'.
+	fname := strings.TrimPrefix(seccompProfile, "localhost/")
 	file, err := ioutil.ReadFile(filepath.FromSlash(fname))
```

Comment:
Can we also add validation that `fname` is absolute, and return an error if it is not?

Reply:
something like this
```
import "path/filepath"
if !filepath.IsAbs(fname) {
    return nil, fmt.Errorf("seccomp profile path must be absolute, but got relative path %q", fname)
}
```

Reply:
@mtaufen Good catch. Will add.